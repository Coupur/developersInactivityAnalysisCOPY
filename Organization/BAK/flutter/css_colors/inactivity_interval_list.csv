abarth,508,509,0.003929273084479371
gspencergoog,1,1.0
