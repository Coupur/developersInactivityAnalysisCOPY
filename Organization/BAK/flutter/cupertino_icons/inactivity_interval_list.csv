xster,108,89,200,0.025
devoncarew,1,1.0
bcko,4,36,41,0.07317073170731707
Hixie,1,1.0
