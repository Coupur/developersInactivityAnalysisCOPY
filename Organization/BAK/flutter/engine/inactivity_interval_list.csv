johnmccutchan,4,10,11,15,35,6,7,28,2,52,35,16,3,2,13,19,7,42,22,14,11,2,5,2,11,2,3,2,5,2,5,20,7,6,23,12,8,7,3,16,512,0.111328125
abarth,2,2,2,3,2,2,2,3,2,3,2,13,2,3,5,2,2,5,2,4,2,4,3,3,2,4,4,2,2,3,2,3,3,2,3,12,3,2,2,2,3,7,2,5,2,3,2,7,5,7,2,7,3,4,7,4,5,2,2,2,9,3,6,3,2,3,5,3,3,2,3,2,10,3,3,2,4,4,3,32,2,4,63,4,10,12,10,8,7,3,2,6,13,17,3,64,5,2,3,9,5,2,12,8,13,2,32,29,28,19,4,14,3,5,16,5,5,41,5,36,218,98,1402,0.2603423680456491
collinjackson,2,2,2,14,4,3,2,16,3,2,4,8,2,8,3,2,2,3,5,6,2,3,5,2,45,4,6,6,2,11,18,7,2,11,13,41,62,146,6,57,47,4,104,729,0.102880658436214
mpcomplete,4,5,2,3,2,4,2,19,8,13,7,15,3,2,5,2,6,33,28,2,13,22,16,60,10,2,75,175,553,0.07775768535262206
eseidelGoogle,3,3,4,3,4,6,97,78,13,10,72,42,357,0.09523809523809523
Hixie,3,23,2,2,3,4,2,3,2,3,3,2,2,3,2,4,4,2,11,2,16,5,4,4,9,2,3,3,3,11,4,5,4,6,2,26,18,8,17,3,17,26,8,5,6,6,35,51,14,4,9,7,4,43,76,14,9,2,3,5,6,15,11,6,26,9,26,5,10,8,11,24,11,3,33,14,13,6,2,4,2,62,25,46,14,144,39,30,50,1329,0.1384499623777276
HansMuller,3,3,6,2,3,3,2,3,5,2,5,10,3,3,3,2,5,5,4,2,3,279,182,41,749,1364,0.04472140762463343
chinmaygarde,3,3,5,2,4,2,2,3,2,3,2,5,3,4,3,3,3,3,7,4,3,4,2,2,2,2,2,3,6,5,5,5,14,6,3,4,3,2,2,3,6,2,3,2,3,3,4,2,3,3,3,4,28,3,2,2,2,6,4,3,12,2,6,19,5,2,5,3,3,2,5,6,3,2,3,6,3,4,4,2,2,2,4,2,14,6,5,3,3,5,3,5,3,3,4,3,20,6,9,5,5,7,2,3,2,5,2,10,2,7,5,2,3,2,2,3,3,4,4,3,2,2,2,10,2,21,3,8,35,4,4,2,2,3,9,5,2,3,4,23,3,8,5,6,3,2,2,4,8,24,91,4,28,2,3,2,2,3,2,4,2,2,2,5,4,4,2,11,3,6,26,2,6,9,3,2,6,2,4,4,3,2,14,4,6,11,4,2,20,10,2,42,6,5,4,6,4,3,6,5,6,2,2,4,3,4,2,2,3,5,2,8,2,2,3,3,14,5,4,3,3,2,1431,0.3081761006289308
qsr,513,174,42,125,855,0.005847953216374269
raphlinus,8,14,15,9,4,22,14,14,5,14,11,8,16,13,7,2,27,6,3,2,39,4,269,0.12639405204460966
vlidholt,3,4,3,4,5,3,4,9,5,2,3,4,3,3,3,2,3,5,107,0.5327102803738317
iansf,4,6,2,2,9,3,2,2,5,8,6,16,81,0.345679012345679
jamesr,7,4,9,8,4,3,20,3,14,4,24,90,9,63,91,26,2,7,7,16,2,7,8,125,5,3,9,26,106,52,193,12,973,0.047276464542651594
jimsimon,1,1.0
tonygentilcore,170,97,269,0.01486988847583643
asankah,1,1.0
jimbeveridge,4,22,48,7,2,85,0.08235294117647059
roozbehp,2,3,65,5,77,71,14,55,77,186,5,5,4,6,8,13,3,607,0.04118616144975288
ianloic,330,111,4,234,4,23,11,2,27,7,7,2,5,6,57,135,967,0.01861427094105481
apwilson,3,15,5,3,8,14,4,2,990,1046,0.010516252390057362
eseidel,1,1.0
afandria,24,25,0.08
pylaligand,212,202,3,45,3,2,7,113,5,10,6,4,3,2,2,6,4,41,3,2,71,19,14,6,5,5,6,8,14,42,28,23,15,19,46,7,296,1311,0.03737604881769641
devoncarew,9,10,23,127,3,29,10,19,13,28,631,906,0.016556291390728478
alhaad,5,189,2,713,14,8,2,937,0.011739594450373533
nona-google,7,15,7,6,12,6,6,2,19,2,12,5,3,3,6,14,15,3,8,15,5,2,5,4,28,6,4,4,42,3,6,5,2,7,5,2,8,3,3,95,7,13,8,2,5,2,10,9,15,8,5,5,16,5,3,3,8,6,10,4,6,2,11,10,600,0.14333333333333334
mishadynin,11,12,0.16666666666666666
emembrives,2,1.0
krisgiesing,16,15,2,3,3,4,2,3,6,4,8,2,4,13,683,5,785,0.035668789808917196
sethladd,65,14,48,111,140,189,16,584,0.0136986301369863
sgraham,2,1.0
ksimbili,1,1.0
jason-simmons,3,3,4,8,3,6,3,3,4,5,14,4,3,21,3,5,4,2,3,4,2,7,5,5,2,3,2,3,5,2,3,8,3,4,4,3,3,5,2,3,6,8,3,23,9,11,6,2,3,2,7,5,6,2,4,5,2,6,9,9,5,3,4,6,6,4,5,3,3,15,6,5,3,27,2,3,4,3,2,2,2,5,2,7,4,13,5,2,2,2,2,3,3,2,2,3,14,13,12,13,2,6,2,3,2,4,2,11,5,2,4,3,4,2,4,2,4,3,3,3,3,5,4,6,3,2,2,3,3,5,3,2,3,2,5,8,4,3,13,7,2,3,2,17,5,6,3,3,4,3,2,4,2,4,6,6,6,3,3,3,4,2,5,2,4,2,6,3,3,3,4,4,17,2,2,6,4,2,3,2,7,7,2,3,7,2,2,3,8,3,3,5,8,6,2,7,3,2,12,2,5,2,6,22,2,2,22,7,21,3,7,8,6,6,4,6,3,8,6,5,2,6,11,8,4,2,18,3,4,7,7,5,2,3,2,1372,0.3097667638483965
j9brown,3,169,202,4,17,9,10,3,4,251,45,203,13,938,0.019189765458422176
ksk9687,8,12,112,16,149,0.03355704697986577
mdakin,3,3,7,0.42857142857142855
tvolkert,106,3,23,3,15,35,10,3,16,7,5,3,3,9,51,6,64,32,6,157,33,4,8,26,176,40,54,4,17,2,7,4,3,74,3,89,11,8,2,19,20,39,69,50,1331,0.042073628850488355
mxia,1,1.0
aghassemi,2,1.0
floitschG,19,8,8,825,89,951,0.007360672975814932
kulakowski,11,3,5,553,575,0.01217391304347826
turnidge,1,1.0
pq,1,1.0
domesticmouse,1,1.0
appsforartists,1,1.0
dpa-g,2,23,26,0.11538461538461539
agampe,1,1.0
stephenhines,1,1.0
yjbanov,19,2,7,8,43,11,10,19,2,388,39,3,64,8,98,12,11,6,30,30,4,5,20,841,0.029726516052318668
rmacnak-google,77,28,42,11,8,56,37,4,15,16,60,2,19,2,42,11,7,7,36,7,12,11,5,4,6,2,37,6,13,5,13,6,29,29,9,20,13,18,3,4,16,16,3,7,15,13,5,2,6,6,7,25,2,27,4,46,2,3,10,4,17,17,153,1155,0.06926406926406926
DanAlbert,1,1.0
ppiet,156,157,0.012738853503184714
mmclenna,1,1.0
adambuchbinder,45,46,0.043478260869565216
cbracken,187,2,4,2,3,4,3,3,2,16,19,2,18,4,3,14,9,3,7,2,3,9,3,2,12,24,38,2,37,5,12,6,7,5,11,9,6,2,3,6,39,2,9,3,10,5,15,5,12,6,49,7,4,9,4,15,73,61,5,13,2,2,2,19,2,4,6,9,6,2,7,5,3,2,4,2,9,10,6,8,12,23,7,6,11,2,6,1078,0.12337662337662338
dragostis,26,21,49,0.08163265306122448
zanderso,46,80,28,34,6,26,80,5,2,34,34,17,10,25,6,5,2,3,9,6,4,2,5,9,11,9,12,7,14,9,3,7,3,10,16,6,6,10,34,25,2,7,67,17,11,14,100,30,23,21,2,2,4,9,6,8,14,34,1042,0.06621880998080615
mikejurka,35,2,40,128,34,33,2,25,52,28,64,3,8,106,14,5,105,27,719,0.03616133518776078
chih-hung,2,1.0
danw,88,5,95,0.042105263157894736
enh,2,3,0.6666666666666666
vosst,1,1.0
petrhosek,258,29,24,178,8,145,643,0.01088646967340591
sigurdm,366,5,30,12,133,20,15,583,0.015437392795883362
HalCanary,1,1.0
yyoon,1,1.0
pombredanne,1,1.0
colincross,6,137,8,153,0.032679738562091505
danafergan,1,1.0
teisenbe,1,1.0
mit-mit,768,73,842,0.0035629453681710215
amirh,285,2,14,3,2,4,2,3,28,18,2,6,5,162,8,2,5,5,2,2,6,4,2,8,2,4,4,24,22,11,2,3,2,2,3,2,3,15,30,49,31,4,2,5,49,15,2,3,9,895,0.07374301675977654
siyamed,8,9,0.2222222222222222
goderbauer,4,8,25,3,2,3,5,20,45,4,7,3,4,3,5,10,4,8,5,5,16,23,3,4,4,4,6,7,4,16,4,2,6,34,5,8,13,2,5,3,5,5,2,4,10,3,16,4,7,2,4,29,3,5,21,5,2,6,46,2,3,7,6,25,4,5,4,14,11,9,3,7,5,4,2,4,5,7,13,2,34,54,19,801,0.14232209737827714
xster,5,62,2,34,57,11,14,8,20,42,8,13,16,83,5,29,21,13,338,785,0.02929936305732484
mattsarett,92,19,4,116,0.034482758620689655
danrubel,5,6,0.3333333333333333
vardhan-cr,11,12,0.16666666666666666
poitch,27,50,5,9,93,0.06451612903225806
mravn-google,5,7,4,5,7,9,9,3,5,10,37,139,11,50,7,41,9,16,32,49,462,0.05844155844155844
jakobr-google,6,22,29,0.10344827586206896
brianosman,101,2,21,78,2,35,4,20,3,10,17,8,7,4,22,16,14,11,6,2,6,2,6,32,59,55,125,7,17,59,15,771,0.04669260700389105
fmalita,35,36,0.05555555555555555
freiling,4,4,22,23,21,3,16,7,46,528,31,21,731,0.023255813953488372
roubert,14,15,0.13333333333333333
szakarias,27,3,12,49,20,49,57,5,2,22,3,44,5,11,5,316,0.05379746835443038
GaryQian,3,5,3,3,2,5,2,2,2,3,4,2,3,384,5,2,4,9,3,7,8,5,2,5,7,4,17,12,2,8,12,12,16,13,6,2,4,3,5,5,39,18,5,14,5,2,10,2,747,0.13253012048192772
aam,3,7,12,8,17,16,11,7,7,2,6,6,4,2,3,6,3,8,14,13,6,3,20,6,6,7,5,2,7,10,2,5,4,5,3,4,3,4,2,10,6,10,5,9,5,13,9,10,14,14,6,7,23,20,23,5,6,7,35,4,7,33,48,46,29,29,725,0.13655172413793104
mehmetf,210,145,110,23,13,2,10,107,94,717,0.016736401673640166
gspencergoog,2,102,3,42,68,13,13,77,18,12,7,2,130,91,44,95,721,0.024965325936199722
ankurmittal,1,1.0
konkers,91,4,96,0.03125
B3rn475,7,12,4,11,3,6,6,8,5,2,66,0.18181818181818182
abhishekamit,49,121,7,179,0.027932960893854747
mraleph,131,3,62,13,13,31,42,97,44,95,2,89,3,626,0.022364217252396165
alexmarkov,58,36,23,29,37,3,16,39,52,66,40,3,8,109,527,0.04174573055028463
jbauman42,197,155,354,0.011299435028248588
schwa423,529,530,0.0037735849056603774
tbmbob,1,1.0
a-siva,14,44,35,19,23,13,15,5,3,12,13,33,79,13,22,14,7,7,28,4,37,5,9,64,2,2,30,50,11,6,625,0.0576
5u3it,1,1.0
mkustermann,5,23,159,12,67,4,184,457,0.02188183807439825
ethanblake4,1,1.0
bkonyi,5,2,4,27,36,42,28,16,14,5,13,30,13,2,9,2,27,13,2,4,3,24,20,50,2,2,6,2,5,2,2,5,5,4,8,5,4,2,2,2,2,4,4,2,5,2,5,2,6,4,2,3,560,0.21785714285714286
bsalomon,12,55,36,7,2,264,377,0.01856763925729443
najeira,223,207,431,0.0069605568445475635
sivachandra,37,38,0.05263157894736842
crelier,47,30,47,42,65,43,275,0.025454545454545455
ErikCorryGoogle,48,49,0.04081632653061224
kevinlindkvist,525,526,0.0038022813688212928
jensjoha,2,21,22,209,7,262,0.022900763358778626
alibitek,1,1.0
Coder-256,1,1.0
jacob314,2,1.0
cpizano,1,1.0
reed-at-google,328,7,72,5,20,434,0.016129032258064516
egdaniel,4,39,2,4,49,11,32,6,148,0.060810810810810814
jvanverth,1,1.0
raggi,1,1.0
timsneath,1,1.0
jonahwilliams,24,2,5,7,7,3,4,48,6,5,2,2,2,2,5,6,5,12,7,10,6,14,3,3,4,3,13,76,23,4,18,8,2,6,3,3,6,34,5,12,6,2,6,2,7,454,0.14537444933920704
skia-flutter-autoroll,10,20,3,3,2,3,2,2,2,2,2,7,2,3,2,3,2,2,2,2,456,0.8771929824561403
sbaranov,67,4,2,41,7,15,77,7,38,2,6,6,5,6,6,23,4,4,5,59,393,0.0737913486005089
dnfield,11,16,21,23,2,104,3,7,12,7,3,2,6,2,3,5,28,19,3,3,2,3,2,4,9,3,7,9,2,3,6,13,4,17,2,6,2,5,2,13,5,2,3,7,448,0.18080357142857142
alanrussian,1,1.0
luigi-rosso,1,1.0
rmistry,8,4,18,0.4444444444444444
majido,1,1.0
pulyaevskiy,1,1.0
xqwzts,1,1.0
cary-clark,17,10,28,0.10714285714285714
matanlurey,1,1.0
liyuqian,5,6,19,10,3,7,3,5,2,3,3,5,2,7,5,17,5,3,13,4,2,5,2,5,2,6,6,12,2,7,21,6,16,7,4,2,2,5,7,5,2,10,2,12,2,3,2,3,9,2,4,3,7,3,3,18,6,10,6,3,2,5,397,0.22921914357682618
ritesh-malav,1,1.0
slightfoot,104,105,0.01904761904761905
bparrishMines,4,17,22,0.13636363636363635
ds84182,1,1.0
bungeman,299,26,326,0.009202453987730062
matthew-carroll,10,59,113,45,2,4,4,4,2,4,4,4,2,2,5,6,6,3,3,20,9,3,8,5,4,2,5,6,362,0.1270718232044199
Phrohdoh,1,1.0
sjindel-google,20,61,21,16,111,5,235,0.029787234042553193
mimers,1,1.0
joshuaseaton,51,52,0.038461538461538464
nathanrogersgoogle,11,55,127,6,41,13,254,0.027559055118110236
jamesderlin,2,12,3,13,21,21,74,0.10810810810810811
aartbik,41,14,24,80,0.05
keertip,1,1.0
NikoYuwono,1,1.0
jslavitz,106,39,146,0.02054794520547945
nichtverstehen,23,5,29,0.10344827586206896
mklim,13,17,3,2,8,28,37,47,4,25,19,20,16,10,32,283,0.06007067137809187
endpress,1,1.0
kangwang1988,8,6,46,65,4,20,42,16,50,258,0.03875968992248062
kzietek,1,1.0
hugovdm,38,161,200,0.015
rafaelring,78,79,0.02531645569620253
zmbush,1,1.0
nightwolf-chen,1,1.0
SupSaiYaJin,248,249,0.008032128514056224
wsxyeah,1,1.0
matthewtsmith,1,1.0
clarkezone,1,1.0
Maxim-Mazurok,1,1.0
Rusino,4,16,5,30,56,0.08928571428571429
justinmc,8,19,2,70,27,8,136,0.058823529411764705
gamebox,1,1.0
jimmycasey,1,1.0
dguralnick,1,1.0
lext-7,1,1.0
long1eu,1,1.0
stuartmorgan,2,8,11,8,6,13,7,4,2,6,2,4,19,2,8,3,2,3,4,2,10,6,140,0.21428571428571427
iskakaushik,5,38,4,24,13,5,8,22,6,2,5,5,140,0.10714285714285714
yunyu,1,1.0
ashawn,1,1.0
ivk1800,1,1.0
HelloCore,7,8,0.25
cyanglaz,13,8,7,37,20,4,92,0.09782608695652174
willlarche,26,27,0.07407407407407407
kf6gpe,1,1.0
arbreng,24,2,15,41,9,92,0.06521739130434782
franciscojma86,86,5,92,0.03260869565217391
Piinks,3,64,68,0.04411764705882353
christopherfujino,1,1.0
johnsonmh,1,1.0
agd,1,1.0
dballesteros7,1,1.0
Qxyat,1,1.0
shihaohong,1,1.0
josephlr,1,1.0
mdempsky,9,15,5,33,0.21212121212121213
dkwingsmt,1,1.0
stupidfive,1,1.0
blasten,14,16,0.1875
terrylucas,1,1.0
zhongwuzw,1,1.0
jmagman,1,1.0
gaaclarke,4,2,7,0.42857142857142855
ojhunt,1,1.0
chunhtai,1,1.0
