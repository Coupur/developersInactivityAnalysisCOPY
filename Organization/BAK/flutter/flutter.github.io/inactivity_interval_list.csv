Hixie,1,1.0
kevmoo,5,7,0.42857142857142855
