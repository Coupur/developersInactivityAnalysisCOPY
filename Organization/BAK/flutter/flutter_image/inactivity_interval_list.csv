yjbanov,19,8,40,12,80,0.0625
zanderso,468,470,0.006382978723404255
santiagofm,1,1.0
xster,1,1.0
