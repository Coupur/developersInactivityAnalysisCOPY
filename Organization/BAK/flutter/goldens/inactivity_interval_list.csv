tvolkert,13,14,0.14285714285714285
Hixie,23,20,25,160,22,22,275,0.03272727272727273
xster,2,22,4,9,19,169,63,63,353,0.028328611898016998
jason-simmons,1,1.0
HansMuller,3,5,0.6
liyuqian,36,2,10,4,13,9,29,5,26,67,50,6,32,13,308,0.06493506493506493
matthew-carroll,1,1.0
goderbauer,70,54,72,198,0.025252525252525252
aam,4,5,0.4
cbracken,237,239,0.012552301255230125
doxuanviet1996,1,1.0
amirh,1,1.0
nataliesampsell,1,1.0
mkustermann,1,1.0
jacob314,7,8,0.25
GaryQian,4,33,8,23,60,15,7,26,38,5,9,2,2,29,5,272,0.07720588235294118
mklim,1,1.0
jonahwilliams,1,1.0
alexmarkov,1,1.0
johnsonmh,83,50,6,3,24,167,0.03592814371257485
mraleph,1,1.0
beckler,2,1.0
Salby,1,1.0
dnfield,13,28,16,6,4,36,105,0.0761904761904762
rami-a,2,6,33,42,0.09523809523809523
jslavitz,6,8,0.375
shihaohong,2,1.0
LongCatIsLooong,52,53,0.03773584905660377
justinmc,2,1.0
dkwingsmt,6,7,0.2857142857142857
Piinks,2,1.0
